ubuntudesign/www.ubuntu.com
# Move partners feeds to the backend

Building on @WillMoggridge's work in #912, this loads all the partners feeds in the backend, so that we can finally switch on HTTPS.

There are now two template tags, `get_json_feed` and `get_rss_feed`, for pulling down either feed in the backend and passing the entries through to the template.

I've also removed `partners.js` as it's no longer used anywhere.

QA
---

Check the feeds in the following pages against live:

- <http://127.0.0.1:8001/server/hyperscale>
  - 'A SELECTION OF OUR HARDWARE PARTNERS'
  - 'A SELECTION OF OUR SOFTWARE PARTNERS'
- <http://127.0.0.1:8001/server>
  - 'A SELECTION OF OUR PUBLIC CLOUD PARTNERS'
  - 'A SELECTION OF OUR BIG DATA PARTNERS'
- <http://127.0.0.1:8001/internet-of-things/partners> - 'A GROWING LIST OF PARTNERS'
- <http://127.0.0.1:8001/cloud/partners> - 'A SELECTION OF OUR CLOUD PARTNERS'
- <http://127.0.0.1:8001/desktop/partners> - 'A SELECTION OF OUR DESKTOP PARTNERS'

I think in the cases where there are two feeds per page, they seem pretty flaky on live, sometimes the wrong feed loads in the wrong place. This should fix that.

Also just look around the site and check everything works, no JS errors, see if you can see any feeds I missed.
